AmazonChimeSDKDemo/AmazonChimeSDKDemo/MeetingViewController.swift

```diff
@@ -278,19 +278,34 @@ class MeetingViewController: UIViewController {
         guard let meetingModel = meetingModel else {
             return
         }
-        let optionMenu = UIAlertController(title: nil, message: "Choose Audio Device", preferredStyle: .actionSheet)
 
+        let optionMenu = UIAlertController(title: nil, message: "Choose Audio Device", preferredStyle: .actionSheet)
+        let currentAudioDevice = meetingModel.currentAudioDevice
         for inputDevice in meetingModel.audioDevices {
-            let deviceAction = UIAlertAction(title: inputDevice.label,
+            var label = inputDevice.label
+            if let selectedAudioDevice = currentAudioDevice {
+                if selectedAudioDevice.label == inputDevice.label {
+                    label = "\(label) ✔︎"
+                }
```

Comment:
Not sure if we want to modify a model property for UI display purposes. It would be better to modify UI using UI elements such as labels

Reply:
nvm we're only modifying the title of the UIAlertAction

Reply:
This doesn't modify property. It will get label from the device either add checkmark or not and add to the UIAlert.